src/one_dragon/base/controller/pc_screenshot/pil_screencapper.py

```
@@ -0,0 +1,48 @@
+
+        Args:
+            rect: 截图区域
+            independent: 是否独立截图
+
+        Returns:
+            截图数组，失败返回 None
+        """
+        try:
+            img = pyautogui_screenshot(region=(rect.x1, rect.y1, rect.width, rect.height))
+            screenshot = np.array(img)
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)